packages/frontend/src/components/common/PivotTable/TitleCell.tsx

```diff
@@ -1,20 +1,36 @@
 import { TitleFieldValue } from '@lightdash/common';
+import { Box } from '@mantine/core';
 import { FC } from 'react';
-import { useStyles } from './tableStyles';
 
 type TitleCellProps = {
     title: TitleFieldValue;
     getLabel: (fieldId: string | null | undefined) => string;
+    level?: number;
 };
-const TitleCell: FC<TitleCellProps> = ({ title, getLabel }) => {
-    const { classes } = useStyles();
-    const textAlign = title?.titleDirection === 'header' ? 'right' : 'left';
+const TitleCell: FC<TitleCellProps> = ({ title, getLabel, level = 1 }) => {
     const label = getLabel(title?.fieldId);
-    // TODO: @irakli - what's the correct place for conditional styling?
+
+    const isEmpty = label === '';
```

Comment:
In what cases is this label empty? Did you get this during testing?

Reply:
this is little strange but `getLabel` which is `getDefaultColumnLabel` returns an empty string for undefined fieldId.

```
if (fieldId === null || fieldId === undefined) {
    return '';
}
```

I did not want to update `getDefaultColumnLabel` but I can change to `isEmpty = !title?.fieldId`

Reply:
Ah I made that change. In the contexts we used it, it was always expected to get a string from that function. Then we had separate logic to determine if the field was null.

We can leave this!